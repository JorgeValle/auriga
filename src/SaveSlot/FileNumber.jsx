import React from 'react';

// Components should represent one piece of data model
class FileNumber extends React.Component {
  render() {

    // const time = this.props.time;

    return (
      <span className="file-number">
        <i className="fas fa-save"></i>
        &nbsp;1
      </span>
    );
  }
}

export default FileNumber;