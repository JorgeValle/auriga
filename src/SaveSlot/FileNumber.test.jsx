import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import FileNumber from './FileNumber.jsx';

configure({
  adapter: new Adapter()
});

describe('<FileNumber />', () => {

  // We shallow render to make sure changes in children don't cause cascading fails
  it('shallow renders', () => {
    const wrapper = shallow(<FileNumber />);
  });

  // Snapshot matching
  it('matches snapshot', () => {
    const wrapper = shallow(<FileNumber/>);
    expect(wrapper).toMatchSnapshot();
  });

});

