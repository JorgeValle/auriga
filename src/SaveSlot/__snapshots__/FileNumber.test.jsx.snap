// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileNumber /> matches snapshot 1`] = `ShallowWrapper {}`;
